cli/src/factory_ops/unboxing/init

```diff
@@ -304,7 +304,16 @@ echo "Bringing HCTL cluster out of maintenance mode" 2>&1 | tee -a ${LOG_FILE}
 hctl node unmaintenance --all 2>&1 | tee -a ${LOG_FILE}
 
 echo "Ensure cluster is in healthy state" 2>&1 | tee -a ${LOG_FILE}
-ensure_healthy_cluster
+
+ensure_healthy_cluster "false" "false"
```

Comment:
Can we convert all exits in `ensure_healthy_cluster` to return to avoid execution halt with exit and reduce the lines of code?